### PR TITLE
[Docs] Grammatical change / Timer

### PR DESCRIPTION
**Pull request by christopherdro**

### CODE REVIEW

File: docs/Timers.md [hunk 1]
```diff
@@ -71,4 +71,4 @@ var Component = React.createClass({
 });
 ```
 
-We highly recommend never using bare timers and always using this mixin, it will save you from a lot of hard to track down bugs.
+We strongly discourage using bare timers and always recommend using this mixin. This will eliminate a lot of hard tracking down bugs.
```
Comment (ide) [inline docs/Timers.md]: "a lot of hard tracking down bugs." could use rephrasing

**Comment by brentvatne** [comment]
Maybe we can add in parens after bare timers (`setTimeout(...)`) and after this mixin (`this.setTimeout(...)`)? Just to be super-extra-clear. Thoughts @christopherdro?

**Comment by christopherdro** [comment]
@brentvatne Sounds good. 
I just tried narrowing it down even further. I feel like the paragraph above already gave a pretty good breakdown. Feel free make any changes you see fit.

**Comment by brentvatne** [comment]
We strongly discourage using the global `setTimeout(...)` and recommend instead that you use  `this.setTimeout(...)` provided by `react-timer-mixin`. This will eliminate a lot of hard work tracking down bugs, such as crashes caused by timeouts firing after a component has been unmounted.

**Comment by christopherdro** [comment]
:+1:

**Comment by ide** [comment]
That reads nicely. Let's publish it.

**Pull request merged**